packages/visualizations/src/Labels.tsx

```diff
@@ -54,7 +54,7 @@ export const Labels: DiscreteAxialChart<string> = props => {
           <text
             x={metricScale(metric(row))}
             y={categoricalScale(categorical(row))! + bandWidth / 2}
-            dx={4}
+            dx={"0.35em"}
```

Comment:
```suggestion
            dx="0.35em"
```

not a critical thing though